_posts/2016-07-29-npm-install-electron.md

```diff
@@ -0,0 +1,67 @@
+---
+title: npm install electron
+author: zeke
+---
+
+With help from the community, we have taken over the `electron` package
```

Comment:
I really like the second sentence of this paragraph and feel like it should be the first sentence, it conveys all the important news.

Then this first sentence can be removed since it is covered in depth in other sections.